### ISSUE TITLE
[IntelliJ-ThirdParty][A11yMAS][Accessibility][Jaws/NVDA][MAS4.2.1][IntelliJ][Wipro]:Jaws and NVDA are not announcing the name of the search edit box  in "Browse Repositories dialog"

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1474]
**Repro Steps:**

1. Open and Start IntelliJ.
2. Tab to and select File->Settings.
3. In "settings" dialog , tab to and select plugin tree item.
4. Tab to and select "Browse Repositories" button .
5. In "Browse Repositories " dialog , tab to search edit field and observe the Jaws and NVDA behaviour

**Actual Result:**
Jaws and NVDA is not announcing the name of the search edit box

**Jaws behaviour:**
Edit
Type in text

**NVDA Behaviour:**
edit
blank

**Expected Result:**
Jaws and NVDA should announce the name of the edit field
Name should be search

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
OSBuild Version: Version 1709(Os Build 16299.192)
![browse repositories-edit-no name](https://user-images.githubusercontent.com/22956130/36103957-bdd893b4-1036-11e8-8988-52244f0215fa.jpg)
![browse repositories-edit-nvda](https://user-images.githubusercontent.com/22956130/36103960-c0cf21fa-1036-11e8-9bde-18a1750f6278.jpg)
![browse repositories-edit-explorer](https://user-images.githubusercontent.com/22956130/36103962-c36d06fc-1036-11e8-91e5-f2c343239106.jpg)

**Comment by Eskibear** [comment]
Cannot reproduce it.

According to the practice, NVDA cannot read content of any dialog in IntelliJ IDEA 2017.3.4 Community.
A ticket is created to track the issue.

Ref:
https://youtrack.jetbrains.com/issue/IDEA-191325

**Comment by Eskibear** [comment]
And it's out of the toolkit's scope

**Comment by sallyur** [comment]
As mentioned above, in our practice we cannot reproduce the issue, as NVDA cannot read content of IntelliJ IDEA at all.

- Importance of experience: it depends, varies from somewhat important to important.
- Quality of experience: not impacted. Since the Narrator is not experiencing this kind of issues, users can use the Narrator to as an alternative workaround.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue and external issue.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749917
